packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -54,4 +57,24 @@ export class WorkspaceInputDialog extends SingleTextInputDialog {
         // Add the path and icon div before the `inputField`.
         this.contentNode.insertBefore(element, this.inputField);
     }
+
+    /**
+     * Append the human-readable absolute root `path` to the dialog.
+     */
+    protected appendRootName(): void {
```

Comment:
As per the description of #7309, the following method should only apply in the case of a multiple-root workspace.
Currently, even if I have a single-root workspace, the root name is displayed which is superfluous.